src/main/scala/nl.knaw.dans.easy.bagstore/server/DefaultServletComponent.scala

```diff
@@ -23,7 +23,7 @@ import nl.knaw.dans.lib.logging.servlet._
 import nl.knaw.dans.lib.logging.servlet.masked.MaskedAuthorizationHeader
 import org.scalatra.{ Ok, ScalatraServlet }
 
-trait DefaultServletComponent extends DebugEnhancedLogging {
+trait DefaultServletComponent {
   this: BagStoresComponent =>
 
   val defaultServlet: DefaultServlet
```

Comment:
@rvanheest  if we remove the trait here we have to add the trait in BagStoreServerComponent:27

Reply:
I would add it on line 26 of that file